dashboard/src/modules/components/OverviewComponent/common-component.jsx

```diff
@@ -18,6 +18,7 @@ import {
 import {
   CaretDownIcon,
   CheckIcon,
+  FileIcon,
```

Comment:
This apparently needs to be removed, or used ...

Reply:
Removed